internal/command/machine/proxy.go

```diff
@@ -75,7 +75,8 @@ func runMachineProxy(ctx context.Context) error {
 		Ports:            []string{"4280"},
 		OrganizationSlug: orgSlug,
 		Dialer:           dialer,
-		RemoteHost:       "_api.internal",
+		// FIXME: consider using fdaa::3 here
```

Comment:
Nah. There's no win to avoiding the _static_ DNS names, which aren't subject to propagation delays.